rest-api-spec/src/main/resources/rest-api-spec/api/cluster.health.json

```diff
@@ -111,6 +111,10 @@
       "awareness_attribute":{
         "type":"string",
         "description":"The awareness attribute for which the health is required"
+      },
+      "ensure_local_node_commissioned":{
```

Comment:
wondering we could use `ensure_node_commissioned` and only support with `_local` to start with. Later we could extend this to any node_id

Reply:
There could only be two kinds of transport request. One which retrieves information from local cluster state of the node or from leader's state. There's no mechanism which says run this transport request on a specific node id. Hence, I feel this would ALWAYS run with local param only

Reply:
Updating it to ensure_node_commissioned